db/migrations/schema-objects/20221122163818.do.fix-school-audit-trigger.sql

```diff
@@ -0,0 +1,63 @@
+DROP TRIGGER IF EXISTS [mtc_admin].[schoolInsertUpdateAuditTrigger];
+
+go
+
+CREATE OR ALTER TRIGGER [schoolInsertUpdateAuditTrigger] ON [mtc_admin].[school]
```

Comment:
ive tried this locally and get the same error without the DROP first.  Bizarre.

maybe remove the OR ALTER and just stick to the traditional method? as the newer construct is clearly broken.

Reply:
I've scoped the trigger name with the schema and this does the trick, as per https://github.com/jon-shipley/patterns/blob/master/mssql-idempotent-sql.md#Triggers